astroid/node_classes.py

```diff
@@ -644,6 +645,30 @@ def nodes_of_class(self, klass, skip_klass=None):
             for matching in child_node.nodes_of_class(klass, skip_klass):
                 yield matching
 
+    def get_assign_nodes(self):
```

Comment:
I would prefer these to be private. Some comments before them mentioning why they are like this would also help in the future in case someone wonders why these couldn't have been made more DRY